plugin/parser/differ/mysql/differ.go

```diff
@@ -45,7 +45,13 @@ func (*SchemaDiffer) SchemaDiff(oldStmt, newStmt string) (string, error) {
 	if err != nil {
 		return "", errors.Wrapf(err, "failed to parse new statement %q", newStmt)
 	}
-	var diff []ast.Node
+
+	var add []ast.Node
+	var inplaceUpdate []ast.Node
+	var nonInplaceDrop [][]ast.Node
+	var nonInplaceAdd []ast.Node
```

Comment:
what the difference between `add` and `nonInplaceAdd`

Reply:
`Add` means I want to add an object like a column or table.
`NonInplaceAdd` means I cannot update the node in place like index, I should drop it and then add it.
BTW, I think it's hard to define where this order applies, is it between objects or within objects, is the object `stmtNode` or table/column/index?
But for now, this order is still appropriate.